snap/hooks/configure

```diff
@@ -1,4 +1,10 @@
-#!/bin/sh -e
+#!/bin/bash -e
 
-# Just a placeholder, however validation of the proxy.{http,https}
-# configuration could be made here
+vm_backend="$(snapctl get vm-backend)"
+
+if [[ -n $vm_backend && ! $vm_backend =~ (libvirt|qemu) ]]; then
+    echo "\"$vm_backend\" is not a supported VM backend"
+    exit 1
+fi
+
+snapctl restart $SNAP_NAME
```

Comment:
Restart seems brutal, but I suppose there's nothing better we can do.

Reply:
Yep, we have to restart multipassd in order for the changed backend to take effect and this is the only way I know of on how to do that.

Reply:
Fixing #130 will help with the UX of this.

Reply:
Are you concerned that when someone wants to switch backends, for example, from qemu to libvirt and they have running VM's, that the VM's would automatically start using the new backend since they were running with the prior backend?

I added a comment to #130 about how that is going to be resolved, but that won't fix switching backends and I kind of think when switching backends, requiring an explicit start to instances should be required since it's a pretty drastic change in daemon and instance management.